src/gpu/d3d11/SDL_gpu_d3d11.c

```diff
@@ -5247,7 +5247,7 @@ static SDL_GPUTexture *D3D11_AcquireSwapchainTexture(
             D3D11_WaitForFences(
                 (SDL_GPURenderer *)renderer,
                 true,
-                (SDL_GPUFence **)&windowData->inFlightFences[windowData->frameCounter],
+                (SDL_GPUFence *const *)&windowData->inFlightFences[windowData->frameCounter],
```

Comment:
Why do you need a cast here?

Reply:
MSVC warnings. I think I could make this less awkward by changing inFlightFences to use SDL_GPUFence, it should only have to cast from D3D11_Fence once in the code.